udp multicast receive (#150) (#360)
ietf-tapswg/api-drafts: draft-ietf-taps-impl.md

Abort:
: Calling `Abort` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.

## UDP Multicast Receive

Connectedness: Unconnected

Data Unit: Datagram

API mappings for Receiving Multicast UDP are as follows:

Connection Object:

> : Established UDP Multicast Receive connections represent a pair of specific IP addresses and ports.  The "unidirectional receive" transport property is required, and the local endpoint must be configured with a group IP address and a port.

Reviewer: ```suggestion
: Established UDP Multicast Receive connections represent a pair of specific IP addresses and ports.  The "direction" transport property must be set to "unidirectional receive", and the local endpoint must be configured with a group IP address and a port.
```
